pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "AutoOS"
+version = "0.1.0"
+description = "Automated Open Science (AutoOS) provides tools for the automated documentation of (autonomously conducted) experiments for upload within the Open Science Framework. These methods build on community-supported and machine-readable standards for expressing quantitative models of brain function and behavior (e.g., the Model Description Format) and corresponding data sets (e.g., the Brain Imaging Data Structure)."
```

Comment:
That sounds good! I assume we can change the description later on? We might also include tools for translating models (like the ones from the theorist) into English language...